examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,504 @@
+{
```

Comment:
<p>Yeah, I see this nice copy-paste from the docs, but it's still unclear, what kind of "short names" I can use. Is 'lambda' a valid thing? Or maybe 'LLR'? How about 'plateau'? Or is it 'reduceplateau'? Make it clear. And do not forget about original docstring.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>